stable/postgresql/templates/_helpers.tpl

```diff
@@ -295,3 +295,22 @@ imagePullSecrets:
 {{- end }}
 {{- end -}}
 {{- end -}}
+
+{{/*
```

Comment:
Can you try something like the following code?
```
{{/*
Get the readiness probe command
*/}}
{{- define "postgresql.readinessProbeCommand" -}}
{{- if (include "postgresql.database" .) }}
    - pg_isready -U {{ include "postgresql.username" . | quote }} -d {{ (include "postgresql.database" .) | quote }} -h 127.0.0.1 -p {{ template "postgresql.port" . }}
{{- else }}
    - pg_isready -U {{ include "postgresql.username" . | quote }} -h 127.0.0.1 -p {{ template "postgresql.port" . }}
{{- end }}
{{- if contains "bitnami/" .Values.image.repository }}
    - [ -f /opt/bitnami/postgresql/tmp/.initialized ]
{{- end -}}
{{- end -}}
```
If it works, it seems more simple

Reply:
+1

Reply:
+1

Reply:
done @carrodher  @juan131